apps/website/app/(docs)/docs/roam/pages/discourse-context-overlay.md

```
@@ -0,0 +1,42 @@
+
+For example, the following score display denotes that the node participates in 5 discourse relations with other nodes, which can be seen in detail in the discourse context popover (1 informs, 1 supports, 2 consistent with, and 1 sourced by; note that 2 of these relations are [custom relations](./extending-personalizing-graph)).
+
+![](/docs/roam/discourse-context-overlay-score.png)
+
+You can think of this default Discourse Score as a rough sense of how "robust" a given node is (discourse relations are more structured, high-signal relationships) relative to how "important" it is (raw references are often noisy). If you'd like, you can also experiment with different functions of discourse relations to a node by defining your own [Discourse Attributes](./discourse-attributes), and have them show up in the overlay!
+
+## Demo
+
+(old video demo)
+
+https://www.loom.com/share/b3d6094cd14a466081b8aa8495eb6542
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)